### ISSUE TITLE
Thread is blocking 

### ISSUE THREAD

**Issue by clobotorre** [unispeech/asterisk-unimrcp#52]
Using MRCPSynth causes lots of messages like this:
Thread LWP 12815 is blocking 'SIP/trunk_i6_allegro-0000000f', already blocked by thread LWP 25892 in procedure ast_waitfor_nandfds

In addition to filling the log file because it logs around 25 lines per second, I wonder if it cannot be a problem.
[full_LogDebug.log](https://github.com/unispeech/asterisk-unimrcp/files/6264150/full_LogDebug.log)

**Comment by achaloyan** [comment]
These messages do not seem to cause any real problem but they need to be addressed or suppressed in one or the other way. I'll take a closer look, when I can.

**Comment by Tinet-AaronAn** [comment]
@achaloyan is there any update about this?

**Comment by achaloyan** [comment]
No, I unfortunately did have the time to look at this closely, but these messages do not seem to introduce any functional problems.
